src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -31,6 +31,18 @@ namespace System
         // The minimum value that a Byte may represent: 0.
         public const byte MinValue = 0;
 
+        /// <inheritdoc cref="IAdditiveIdentity{TSelf, TResult}.AdditiveIdentity" />
+        public const byte AdditiveIdentity = 0;
```

Comment:
The interface property can be satisfied by a const?

Reply:
Ah, it's the inheritdoc that confused me.  This isn't actually implementing that property.